src/types/fetchers.ts

```diff
@@ -14,10 +14,13 @@ export type CryptoCompareHistoFetcherResult = [
 
 export type CryptoCompareMultiProcessorResult = number | undefined;
 
+export type StringMultiProcessorResult = string | undefined;
+
 export type OnChainDataFetcherResult = string | number;
 
 export type FeedFetcherInterfaceResult =
   | Promise<number | undefined>
+  | Promise<StringMultiProcessorResult[]>
```

Comment:
I think this may need a refactor in the future since every time we add new return types to the interface, not very different at returning `any`.

Reply:
I added a TODO. Good call